### ISSUE TITLE
[UI] Embedded - Fix Authentication

### ISSUE THREAD

**Issue by richard-cox** [epinio/ui#129]
- Embedded UI uses the default user secret within basic auth for api requests
- Default user is now going to be optional (see https://github.com/epinio/helm-charts/issues/214)
- In addition the password in the secret is now bencrpt'd, yet we need it in plain text to make the api requests
- Possible workaround - dashboard automatically creates an epinio-ui user with password in plain text. it will only be usable by the epinio ui and not the cli.
- in the future though the dashboard should handle different creds or epinio uses the rancher auth provider
```[tasklist]
### Tasks
- [ ] https://github.com/epinio/ui/issues/358
- [ ] https://github.com/epinio/ui/issues/359
- [x] Build / publish a new version of the epinio ui extension
```

Note - https://github.com/epinio/ui/issues/360 has been removed from this epic, given that it might be a while until this is achievable

**Comment by juadk** [comment]
When we talked about the broken dashboard, you mentioned that issue to me but is it really related to the fact that we cannot see the epinio icon in the nav menu anymore? (just to be sure).
I can still install epinio through the marketplace but once done, there is no button to access to the epinio instance.

**Comment by juadk** [comment]
ok so we definitively have to wait for that issue to be resolved to progress.

**Comment by thehejik** [comment]
@richard-cox Hello, our CI is still using non-existing Epinio icon/button in the Rancher UI to access Epinio dashboard so the Rancher UI tests are broken then. 

There is a couple of other ways to get the link but most Rancher products offer an icon, so I think it would be nice to have it.

Do you think the lack of icon deserves a separate card to prioritize this issue or should we do a workaround in CI?

**Comment by richard-cox** [comment]
@thehejik To clear a few things up, there are two 'modes' that the UI has
- Standalone - The UI is served up by the UI backend, and is deployed and released as part of the general Epinio release. 
- Embedded - The UI is served up by the Rancher Dashboard and has a separate build/package for the Rancher Management product. This is not currently released (but we hope to in the future) and is blocked on this issue (#129).

An alternative mode would be, as i think you were implying from your comment, is to have links from the Rancher Dashboard to the standalone UI. That is currently out of scope and would be in a different form (the link would appear as part of the cluster explorer area rather than a menu item in the burger menu).

**Comment by richard-cox** [comment]
- local user - possible cloud credentials / secret , use rancher proxy & cloud credential / secret id

**Comment by richard-cox** [comment]
Closing, tested as part of https://github.com/epinio/ui/issues/358
